lib/moleculer/packets/info.rb

```diff
@@ -92,6 +89,12 @@ def as_json
           client:   @client.as_json,
         )
       end
+
+      private
+
+      def get_config(data)
+        OpenStruct.new(Hash[HashUtil.fetch(data, :config).map { |i| [StringUtil.underscore(i[0]), i[1]] }])
```

Comment:
Would this be better understood if it were in a class? This is a super fancy way to make the data behave like an object. If we defined the class we could define the API for the object, rather than let the data define it for us. 
If that API needs to be more dynamic I'm fine leaving this as is.